solr/modules/s3-repository/src/java/org/apache/solr/s3/S3StorageClient.java

```diff
@@ -422,6 +423,7 @@ private Collection<String> deleteObjects(Collection<String> paths) throws S3Exce
    * @param batchSize number of deletes to send to S3 at a time
    */
   @VisibleForTesting
+  @SuppressForbidden(reason = "Lists.partition")
```

Comment:
I'd love to use Java Streams of this but doesn't seem to be a good fit. Will look at it again.